### DISCUSSION TITLE
How to request a new example repo?

### DISCUSSION THREAD

**Discussion by jeffpball** [vercel/next.js#57063] [Help] [answered]
### Summary

Is there a way to make a request to have a new example repo created? 

I am hoping for a Todo app similar to the one in the existing repo "with-aws-amplify" but using App Router instead of Pages Router. Maybe name it "with-aws-amplify-approuter"?

Here is the existing repo: https://github.com/vercel/next.js/tree/canary/examples/with-aws-amplify

**Comment by KryptXBSA** [comment, marked as answer]
I would recommend creating a new GitHub issue in the repository, outlining the details of the new example repository you'd like to see.

**Reply by jeffpball** [reply]
Will do, thanks a lot!
